GUIDELINES.md

```diff
@@ -4,36 +4,58 @@ Note: This is a living document managed by the SDL Project Maintainer and may ch
 ### File Structure
 
 #### Common Library
-Common library files shall be stored in the `/lib/js` directory, grouped into subdirectories by their logic type (e.g. protocol, transport, session, rpc, manager, util), similar to the Java Suite.
+Common library files shall be stored in the `/lib/js/src` directory, grouped into subdirectories by their logic type (e.g. protocol, transport, session, rpc, manager, util), similar to the Java Suite. Compressed distributions of the library shall be written to `/lib/js/dist/SDL.js`.
 
 #### Node.js Library
-Node.js library files shall be stored in the `/lib/node` directory, grouped into subdirectories in the same way as common files.
+Node.js library files shall be stored in the `/lib/node/src` directory, grouped into subdirectories in the same way as common files.
 
 #### Example Projects
 Example projects shall be located in the `/examples/js` and `/examples/node` directories, respectively.
 
 
 ### Object-oriented Programming
 - Concepts must be broken down into classes for an object-oriented programming methodology.
-- Although ECMAScript does not yet officially support private properties, properties shall be named with an underscore prefix to avoid conflict with getters and setters and to allow for an easy transition to private properties when they become available.
-- Rather than promoting the direct manipulation of property values, the use of prefixed `get` and `set` methods shall be used. For example: `setMajor(major)` and `getMajor()`.
+- Although ECMAScript does not yet officially support private properties, properties shall be named with an underscore prefix to allow for an easy transition to private properties when they become available.
+- Rather than promoting the direct manipulation of property values, the use of prefixed `get` and `set` methods shall be used. For example: `setMajor(major)` and `getMajor()`. The use of class "getters" and "setters" shall be avoided.
```

Comment:
```suggestion
- Rather than promoting the direct manipulation of property values, the use of prefixed `get` and `set` methods shall be used. For example: `setMajor(major)` and `getMajor()`. The use of class "getters" and "setters" shall be avoided. For example `version._major = 1`
```

Reply:
Just think it would be good to be add an example of what not to do.